src/tcgwars/logic/impl/gen7/ShinyVault.groovy

```diff
@@ -0,0 +1,398 @@
+package tcgwars.logic.impl.gen;
```

Comment:
Change this to `package tcgwars.logic.impl.gen7;`

CI won't build unless this is set